views/extras.tmpl

```diff
@@ -72,11 +79,38 @@
 {{define "footer"}}
 <footer class="navbar-fixed-bottom">
     <div class="container wrapper text-center">
-        <a class="nav-item" href="dcrdata.org">dcrdata.org</a>
+        <a class="nav-item" href="https://dcrdata.org">dcrdata.org</a>
```

Comment:
The other PR might be first on this line.  But anyway I put up www.dcrdata.org as a special page for the project